aha-table.html

```diff
@@ -429,15 +429,23 @@
       var scrollBody = Polymer.dom(aha.root).querySelector('.scroll-body.aha-table');
       var rowHeight = 42;
 
+      var closestPageSize;
```

Comment:
Since you add an event listener in `attached`, you should also remove that event listener in `detached`.